### DISCUSSION TITLE
Hi~ My student application has been approved which was submitted on Jul 8th. However, I have not received my education benefits. Could you please assist me in resolving this matter? Thank you very much for your help.

### DISCUSSION THREAD

**Discussion by liji20030124** [community/community#132025] [GitHub Education]
### Hi everyone,

Hi~ My student application has been approved which was submitted on Jul 8th. However, I have not received my education benefits. Could you please assist me in resolving this matter? Thank you very much for your help.

**Comment by Adya-Prasad** [comment]
haha
I just answered this question from another user
and seen your question below

let me help you dear

I've included the section that I feel would best assist.
[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

If You still have any concerns! Mark my comment as an answer and reply to me with your concern. I would love to help you ahead
